suite-common/formatters/src/tests/cryptoAmountFormatter.test.ts

```diff
@@ -0,0 +1,41 @@
+import { PROTO } from '@trezor/connect';
```

Comment:
name of file should be `prepareCryptoAmountFormatter.test.ts`

Reply:
fixed